### PR TITLE
General updates unai - General page and AZ exam pages

### PR DESCRIPTION
**Pull request by unaihuete93**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ sidebar:
 
 :earth_americas: [Microsoft Docs](https://www.aka.ms/Docs "Microsoft Docs")
```
Comment (mkennten) [inline index.md:91]: https://www.aka.ms/Docs points to Microsoft Learn directly, I suggest changing this either to https://learn.microsoft.com/en-us/docs/ or remove it.

**Review by mkennten — COMMENTED** [review]
LGTM, only one minor thing to resolve, see comment

File: index.md (hunk 1 above)
Comment (unaihuete93) [reply index.md:91]: lets remove it :)

Comment (unaihuete93) [reply index.md]: done, will merge PR

**Pull request merged**